inc/widgets/author-bio.php

```diff
@@ -25,28 +25,79 @@ function __construct() {
 	function widget( $args, $instance ) {
 
 		global $post;
-		$author_id = $post->post_author;
 
-		$output = '';
+		$authors = array();
+		$bios = '';
 
-		echo $args['before_widget'];
+		if ( is_singular() ) {
+			if ( function_exists( 'get_coauthors' ) ) {
+				$authors = get_coauthors( get_queried_object_id() );
```

Comment:
is there a compelling reason to use `get_queried_object_id()` here (and continuing below) rather than `$post->ID` ?